src/components/ModalDelete.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import * as Dialog from '@radix-ui/react-dialog';
+import Trash from '../assets/trash.svg';
+
+function ModalDelete() {
```

Comment:
O componente poderia ser parametrizado.

```suggestion
function ModalDelete({ 
    text, 
    onCommit,
    confirmationTitle="Confirmar Ação", 
    cancelButtonText="Cancelar", 
    confirmationButtonText="Confirmar" 
    }) {
```

Assim seria melhor de reutilizar ele, mesmo em casos onde a gente precise utilizá-lo para confirmação de outras coisas que não seja uma exclusão genérica. Como a maioria das props tem um valor padrão, o componente continua sendo fácil de usar e fica muito mais flexível.

Também é necessário adicionar a função de será executada em caso de confirmação por parâmetro. No caso dos modais de confirmação de exclusão, a função de callback `onCommit` será a responsável pela chamada da API para exclusão.

Reply:
:+1: